### ISSUE TITLE
Make sure to use bash

### ISSUE THREAD

**Issue by mrtndwrd** [aslafy-z/helm-git#10]
In my setup (Ubuntu 18.04), `/usr/bin/env sh` uses `dash`, which does not support the `-o pipefail` option. This can be solved by using `/usr/bin/env bash` instead. I'll make a PR in a minute.

**Comment by mrtndwrd** [comment]
FYI this is the error I'm getting: 

```
.helm/plugins/helm-git.git/helm-git: 7: set: Illegal option -o pipefail
```

@aslafy-z please take a look, I can't use this plugin without #11 merged into it.

**Comment by aslafy-z** [comment]
Thank you for the heads-up. The goal was to keep the script 100% posix shell compliant so I'm not sure if this PR makes sense. I think I'd better protect pipes from breaking out instead of switching interpreter. Do you have any recommendations towards that goal?

**Comment by mrtndwrd** [comment]
I don't have a lot of experience with POSIX compliance, so I can't really recommened anything... Why do you *need* pipefail? At first glance I don't see a lot of places where the command before the pipe can really fail. 

If you only have to rewrite a few lines, I think I would go for something like [this](https://stackoverflow.com/a/54931544). Let me know if there's anything I can do without having to know the script from top to bottom ;)

**Comment by aslafy-z** [comment]
Closing as #8 fixes it

**Comment by mumoshu** [comment]
Thanks @aslafy-z ☺️

**Comment by ariep** [comment]
This appears not completely fixed in current master. I have the same environment as mrtndwrd. In the file `helm-git-plugin.sh` the `-o pipefail` was removed, but not in the wrapper executable `helm-plugin`. I still get the same error message when I try to use helm-git, and it's fixed by changing the shebang.

**Comment by aslafy-z** [comment]
@ariep Thanks for reporting, it has been fixed in `v0.4.2`.
